fineract-provider/build.gradle

```diff
@@ -63,10 +63,10 @@ task prepareInputYaml {
 rat.dependsOn prepareInputYaml
 
 resolve {
-    logging.captureStandardOutput LogLevel.INFO
+    // logging.captureStandardOutput LogLevel.INFO
```

Comment:
You wanna uncomment or remove?

Reply:
Good catch... have to add again, was just temporarily commented out, because of some random Gradle exception in that task.